---
layout: consultancy
title: Consultancy
permalink: /consultancy/
---

## Services

<div style="font-size: 0.9em;">I have been actively collaborating with industry partners for several years and am happy to receive any new inquiries. Feel free to contact me at ryan.m.imrie@gmail.com with any questions.</div>

#### Assay Design, Wet-Lab Workflows, and SOPs

<div style="font-size: 0.9em;">I have many years of experience in molecular biology wet-lab techniques, with a particular focus on improving usability, reliability, and throughput for the following techniques:
<ul>
    <li>DNA & RNA extraction from animal tissue, bacterial and animal cell cultures, and stool samples.</li>
    <li>Single and multiplex conventional PCR, qPCR and qRT-PCR.</li>
    <li>Oxford Nanopore sequencing library preparation and multiplexing</li>
    <li>DNA & RNA targeted amplicon sequencing.</li>
</ul></div>

#### Bioinformatics Analysis and Workflows

<div style="font-size: 0.9em;">I have extensive experience of bioinformatics analysis of next-generation sequencing data, including the design of bespoke solutions for unique projects. While I am happy to receive enquiries about any bioinformatics projects, I can offer expertise in the following:
<ul>
    <li>Illumina and Oxford Nanopore demultiplexing, quality control, and library analysis.</li>
    <li>Reference sequence alignment and assembly.</li>
    <li>De novo genome assembly.</li>
    <li>Genome annotation.</li>
    <li>Metagenomics and meta-transcriptomics analysis.</li>
    <li>RNAseq alignment and expression analysis.</li>
    <li>Multiple sequence alignment and phylogenetic reconstruction.</li>
</ul></div>

#### Bespoke Coding Solutions

<div style="font-size: 0.9em;">Some projects require novel or unique solutions to data collection, wrangling, and analysis. With over a decade of experience coding in Python, R, Bash, and Javascript, I am happy to provide bespoke coding solutions to any projects that require them. This includes the development of web apps and GUIs that can be easily interacted with by non-coders. Some examples of past projects are:
<ul>
    <li>A Python image processing pipeline to isolate and quantify endosomes in 3D from confocal microscopy images.</li>
    <li>A custom Bash script to demultiplex Oxford Nanopore reads prepared with Illumina sequencing barcodes.</li>
    <li>An custom R script to efficiently extract and combine qRT-PCR amplification data from many StepOne Plus Real-Time PCR files.</li>
    <li>An R image processing pipeline to segment and quantify melanisation of <em>Galleria mellonella</em> larvae from raw DSLR images.</li>
</ul></div>

#### Statistical Analysis and Reporting
<div style="font-size: 0.9em;"></div>


## Consultancy Rates

<div style="font-size: 0.9em;">The following are my standard consultancy rates, negotiable for any large or recurring projects. <br><br>
It is common for computer based projects to consist of a few hours of active work followed by a period where the computer is processing data unsupervised. To reflect this, a reduced rate that covers only the cost of running and maintaining the bioinformatics system is used. For example, a straightforward metagenomics analysis of a small number of samples may require 2 hours of active time (£110) plus 6 hours of processing time (£30), for a total of £140.
<br><br>
<table border="1">
    <tr>
        <th>Type of Work</th>
        <th>Hourly Rate</th>
        <th>Daily Rate</th>
        <th>Description</th>
    </tr>
    <tr>
        <td>Hourly/daily work rate</td>
        <td>£55</td>
        <td>£450</td>
        <td>This rate applies to active time spent working on a project. This includes meetings, coding, data management, analysis, and writing.</td>
    </tr>
    <tr>
        <td>Hourly computer processing rate</td>
        <td>£5</td>
        <td>N/A</td>
        <td>This rate applies to the processing time used by the bioinformatics system to complete a project, and covers the costs of running and maintaining the system I use for processing.</td>
    </tr>
</table>
</div>
